agent_payload.proto

```diff
@@ -72,7 +72,10 @@ message SketchPayload {
                         int64 ts = 1;
                         int64 n = 2;
                         double min = 3;  
-                        repeated Entry entries = 4 [(gogoproto.nullable) = false];
+                        double max = 4;
+                        double avg = 5;
+                        double sum = 6;
+                        repeated Entry entries = 7 [(gogoproto.nullable) = false];
```

Comment:
I thought I would keep the min and max together since we're not using this yet, but you're right, of course.